pkg/server/manifest_handler.go

```diff
@@ -29,6 +29,9 @@ func manifestHandler(cfg *Config) http.HandlerFunc {
 
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		w.Header().Set("Content-Type", "application/json")
+		w.Header().Set("Cache-Control", "no-cache, no-store, must-revalidate")
+		w.Header().Set("Pragma", "no-cache")
```

Comment:
I think `Pragma` shouldn't be used now: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Pragma